pyvecorg/data/coc.yml

```diff
@@ -0,0 +1,51 @@
+heading:
+  cs: Code of Conduct
```

Comment:
Pravidla chování?

Reply:
uf! to bude tezky prekladat. pojdme se soustredit na tu ENG verzi.